### PR TITLE
Remove Lisk

### PR DESCRIPTION
**Pull request by matejcik**
Per https://lisk.com/blog/development/announcing-lisk-mainnet-v3-migration, the currently existing Trezor support for Lisk will stop functioning. This PR removes all Lisk-related functionality from Trezor codebases.

I'm keeping the entry in `misc.json` and `support.json` as historical data.

### CODE REVIEW

**Comment by matejcik** [comment]
Cc @szymonlesisz for protobuf changes.

I removed entries from `MessageType`, instead of marking them `deprecated`. Will Connect be able to handle this, or should those entries be kept?

File: legacy/firmware/fsm_msg_common.h [hunk 1]
```diff
@@ -75,7 +75,6 @@ bool get_features(Features *resp) {
   resp->capabilities[1] = Capability_Capability_Bitcoin_like;
   resp->capabilities[2] = Capability_Capability_Crypto;
   resp->capabilities[3] = Capability_Capability_Ethereum;
-  resp->capabilities[4] = Capability_Capability_Lisk;
```
Comment (andrewkozlik) [inline legacy/firmware/fsm_msg_common.h:78]: It seems to me we should reduce the capability count by one and shift the indices of NEM, Stellar and U2F.

Comment (matejcik) [reply legacy/firmware/fsm_msg_common.h:78]: 9c0a16a

**Comment by szymonlesisz** [comment]
> I removed entries from `MessageType`, instead of marking them `deprecated`. Will Connect be able to handle this, or should those entries be kept?

@matejcik i guess it's ok to remove them. i will remove those methods from connect or we will make some placeholder with information about removed support, so those messages will never be called

**Review by tsusanka — APPROVED** [review]
Did not test, but LGTM.

**Review by prusnak — APPROVED** [review]
utACK

**Pull request merged**